### ISSUE TITLE
Dragging tabs no longer shows tabs physically moving

### ISSUE THREAD

**Issue by bsclifton** [brave/browser-laptop#6033]
**Did you search for similar issues before submitting this one?**
YEs

**Describe the issue you encountered:**
Tabs used to show an animation of the surrounding tabs moving out of the way:
![tabs moved](https://cloud.githubusercontent.com/assets/4733304/20907909/5ee6aae0-bb0e-11e6-9526-66d659861183.gif)

Somewhere along the way, this regressed.
Per @bradleyrichter:
This was in 0.12.1 and it fell off shortly after that.  By 0.12.5 it was gone

**Expected behavior:**
Should behave as they do in the picture

- Platform (Win7, 8, 10? macOS? Linux distro?):

- Brave Version: 0.12.5+

- Steps to reproduce:
    1. Drag tab over another tab
    2. Expect to see tabs move out of the way, so you know where it'll be placed
    3. Be disappointed :frowning_face:

**Comment by bsclifton** [comment]
I narrowed it down; great memory, @bradleyrichter :smile: 

This behavior was present in 0.12.1 (https://github.com/brave/browser-laptop/commit/f51a293574f7153392af50887fb1cfe859a098cd)

This behavior was broken by 0.12.2 RC1 (https://github.com/brave/browser-laptop/commit/dfe352c8b491d3bf7f3b7bfbce88c901fe504a09)

**Comment by NejcZdovc** [comment]
I tried to reproduce this on my computer, but when I go back in time I get the following error:

> Module version mismatch. Expected 50, got 49

I tried with `node rebuild` to version 1.4, which solved the problem, but tabs were not behaving as are on your screenshot in version 0.12.1 (f51a293).

**Comment by bsclifton** [comment]
@NejcZdovc interesting- did you clear your node_modules and redo the `npm install`?

I was able to narrow down the above not by using source, but by downloading/installing releases (and working off the commits tied to them)

**Comment by NejcZdovc** [comment]
@bsclifton yes I deleted my `node_modules` even `electron` folder, clean install and even clean pull, but I always get this error. When I switch back to master and run `npm_install` everything was working correctly out of the box.

I am using node version 7.2.0 and npm version 4.0.3. I would expect that I would get a different error something like `expected 50, got 52`, because I was going back to September. But it's other way around which is strange.

**Comment by NejcZdovc** [comment]
I would love to tackle this problem, but I need to somehow make it work first :smiley:

**Comment by bsclifton** [comment]
You may try installing an older version of node.js; I believe back then, we were using version 6.3.  Because our fork of electron involves compiling node.js, it may need to be on the same version

**Comment by NejcZdovc** [comment]
Will try with node version switcher to set it to 6.3, hope this will help

**Comment by jonathansampson** [comment]
This was recently mentioned on Twitter by a user:

> Rearranging pinned tabs feels weird, like they're not really moving. Chrome's animations are a good feedback as I'm dragging them that something is happening. With Brave I just have to trust that letting go will drop the pinned tab where I want it.
> https://twitter.com/jamonholmgren/status/821984857232281600

Dragging tabs in 0.13.0 (Preview 9)

![drag-tab](https://cloud.githubusercontent.com/assets/815158/22097952/6bf6ff8e-dde9-11e6-84b0-df3ee916e39a.gif)

**Comment by bsclifton** [comment]
@NejcZdovc I know it's been a while- did you want to check this one out?

**Comment by luixxiul** [comment]
It seems that it does not work as it should

![dnd](https://cloud.githubusercontent.com/assets/3362943/22318117/77649bec-e3bb-11e6-8516-e90e5de1f904.gif)

the tab should be dropped to where :no_entry_sign: is displayed.

Also we don't need to have this one:

![dnd](https://cloud.githubusercontent.com/assets/3362943/22318210/2adaf1f8-e3bc-11e6-81b2-eb51ffc25eb6.gif)

**Comment by luixxiul** [comment]
I created the issue as a follow up: #6033

**Comment by NejcZdovc** [comment]
@luixxiul regarding the second one (space around active tab) I already talked with @bradleyrichter and I will remove it when I will be working on tab animations

**Comment by luixxiul** [comment]
Sweet!
